API.md

```diff
@@ -9,6 +9,14 @@ in the [API.md](https://github.com/containers/libpod/blob/master/API.md) file in
 
 [func Commit(name: string, image_name: string, changes: []string, author: string, message: string, pause: bool, manifestType: string) string](#Commit)
 
+[func ContainerCheckpoint(name: string, keep: bool, leaveRunning: bool, tcpEstablished: bool) string](#ContainerCheckpoint)
+
+[func ContainerExists(name: string) int](#ContainerExists)
```

Comment:
Why is this better than calling GetContainer() and checking for an error?

Reply:
I don't see this as a Python issue at all, I can call the same thing from nodejs.  Now we have two ways of determining if a container exists.  

If GetContainer(foobar) returns an error constant that image does not exist, we don't need this other interface.

Reply:
yes, we agree